CMS-Enterprise/mint-app
# [MINT-3211] Custom Solutions Form Modal

# MINT-3211

## Description

* Add MTOMilestoneTranslation and all the CH stuff
* Add SolutionForm component, testing file, gql mutation, translations





## How to test this change

1. Navigate to MTO home page
2. Click `Add custom solution` button
3. Verify that modal matches figma and functions as expected
4. Verify error and success messages

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
